bitnami/common/templates/_affinities.tpl

```diff
@@ -77,6 +78,13 @@ preferredDuringSchedulingIgnoredDuringExecution:
           {{- range $key, $value := $extraMatchLabels }}
           {{ $key }}: {{ $value | quote }}
           {{- end }}
+      {{- if $namespaces }}
+      namespaces:
+        - {{ .context.Release.Namespace }}
+        {{- with $namespaces }}
+        {{ include "common.tplvalues.render" (dict "value" . "context" $) | nindent 8 }}
+        {{- end }}
+      {{- end }}
```

Comment:
As you are adding the Release.Namespace by default, what about if we name the variable "extraNamespaces"?

```suggestion
      {{- if $extraNamespaces }}
      namespaces:
        - {{ .context.Release.Namespace }}
        {{- with $extraNamespaces }}
        {{ include "common.tplvalues.render" (dict "value" . "context" $) | nindent 8 }}
        {{- end }}
      {{- end }}
```

Reply:
I agree, this is a clearer example

Reply:
@jotamartos thank you for review,
I made the suggested changes